modules/application/src/main/java/edu/northeastern/truthtree/controller/basicInfo/BasicInfo.java

```diff
@@ -108,11 +109,10 @@ public JSONArray getBasicCountiesInfo(@RequestParam
    */
   @Override
   @RequestMapping(value = "/api/cities", method = RequestMethod.GET)
-  public JSONArray getBasicCitiesInfo(@RequestParam
-                                                (value = POPULATION_RANGE, required = false)
-                                                int[] range) {
-    JSONArray response = this.service.getBasicCitiesInfo(range);
-    return response;
+  public ResponseEntity<PageDTO<LocationDTO>> getBasicCitiesInfo(
```

Comment:
Can you rename page to pageNumber? I guess other pagination supported apis have used pageNumber.

Reply:
addressed